dashboard/test/models/user_test.rb

```diff
@@ -4145,38 +4145,34 @@ def hide_scripts_in_sections(section1, section2)
     migrated_teacher = create(:teacher, :with_google_authentication_option, email: email)
 
     migrated_teacher.admin = true
-    migrated_teacher.save!
 
-    assert migrated_teacher.admin
+    assert migrated_teacher.valid?
   end
 
   test 'does set admin to false when it is not a google oauth codeorg account' do
```

Comment:
Can we rename this and similar tests now that we're validating a constraint instead of setting admin to false?  Something like `'cannot grant admin role when not a google oath codeorg account'`

Reply:
Good point!